docs/src/content/docs/docs/components/checkbox.mdx

```diff
@@ -43,14 +43,14 @@ There's three different sizes for checkboxes:
     </PreviewCard>
   </TabItem>
   <TabItem label="Code">
-    ```astro /size="[a-z]+"/ showLinenumbers title="CheckboxSizesExample.astro"
+    ```astro /size='[a-z]+'/ showLinenumbers title="CheckboxSizesExample.astro"
     ---
     import { Checkbox } from '@studiocms/ui/components';
     ---
 
-    <Checkbox label="Small" color="primary" size="sm" defaultChecked />
-    <Checkbox label="Medium" color="primary" defaultChecked />
-    <Checkbox label="Large" color="primary" size="lg" defaultChecked />
+    <Checkbox label='Small' color='primary' size='sm' defaultChecked />
```

Comment:
Question for the whole PR, but is there a reason we've gone for single quotes everywhere? I'm not necessarily against it in code, but I do prefer " for docs as I think it's the "more correct" thing to do

Reply:
It's really just a thing I decided on. Probably something we can revert, although I would prefer that not having to happen before 0.2.0 since it's a lot of work which I don't have the time for (before Friday that is)

Reply:
I think for docs, personally, double quotes reads better. But I'm not fussed about it being before friday, no, it can be next patch!

Reply:
I will note @louisescher and @jdtjenkins Docs changes that are correcting things like punctuation wouldn't need a patch change, since they aren't actually part of the package